media-sound/musescore/musescore-3.0.0_git.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="An open source and free music notation software"
+DESCRIPTION="This aplication is used for creating, playing and printing beautiful sheet music"
+HOMEPAGE="https://musescore.org/"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="8d8ceee1bd208cb2478b91d13ef843bc0980be7b"
+SOURCE_URI="https://github.com/musescore/MuseScore/archive/$srcGitRev.zip"
+CHECKSUM_SHA256="0acf4de2752603595a0943e1d5fa267565df3c7c536a1d4637db6bccd3c0faf5"
+SOURCE_DIR="MuseScore-$srcGitRev"
+PATCHES="musescore-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x_86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+ 	musescore$secondaryArchSuffix = $portVersion
+ 	cmd:mscore
+ 	cmd:musescore
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libogg$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libQt5$secondaryArchSuffix
+	lib:libmp3lame$secondaryArchSuffix
+	lib:libQt5Gui$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
+	lib:libQt5Designer$secondaryArchSuffix
+	lib:libQt5Network$secondaryArchSuffix
+	lib:libQt5Test$secondaryArchSuffix
+	lib:libQt5Quick$secondaryArchSuffix
+	lib:libQt5Xml$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libsndfile$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libQt5$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Designer$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Quick$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc${secondaryArchSuffix}
+	cmd:getconf
+	cmd:make
+"
+BUILD()
+{
+	make release PREFIX=$prefix UPDATE_CACHE=FALSE
+}
+
+INSTALL()
+{
+	make install PREFIX=$prefix UPDATE_CACHE=FALSE
+	mv $prefix/share $prefix/data
+}
+TEST()
+{
+	make check
+}
+
```

Comment:
You have a Control-S in the last line, below the line that has the closing curly brace.
Could you please remove the last line? Thanks!